### DISCUSSION TITLE
@apply to dark mode

### DISCUSSION THREAD

**Discussion by gregoirepuget** [tailwindlabs/tailwindcss#3599] [Help]
Hello,

I think @apply doesn't work with dark mode.

Something like :
`h1 {
    @apply dark:text-white;
  }
  `
  
  i've got issue with something like "class doesn't exist"

**Comment by adamwathan** [comment]
Hey! I'm guessing you probably haven't enabled dark mode in your config file?

```js
module.exports = {
  darkMode: 'media',
  // ...
}
```

Here's a demo that shows it working:

https://play.tailwindcss.com/70ZhNTxIl9
